include/odp/api/spec/packet_io.h

```diff
@@ -489,6 +489,13 @@ typedef union odp_pktout_config_opt_t {
 		 */
 		uint64_t no_packet_refs  : 1;
 
+		/** Enable packet aging and drop
+		 *
+		 * 0: application will not request packet aging (default)
+		 * 1: application will request packet aging
```

Comment:
This could be: 1: application may request packet aging

Other than that, the patch looks ok to me.